2021-12-28 04:21:06,077 (INFO) app_qr.py:qr => IP: 127.0.0.1 GET /qr-code page
127.0.0.1 - - [28/Dec/2021:04:21:06 +0000] "GET /qr-code HTTP/1.1" 200 25095 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:04:21:06 +0000] "GET /static/js/qr.js HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:04:21:06 +0000] "GET /static/assets/images/gappedmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:04:21:06 +0000] "GET /static/assets/images/roundedmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:04:21:06 +0000] "GET /static/assets/images/verticalbarmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:04:21:06 +0000] "GET /static/assets/images/horizontalbarmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:04:21:06 +0000] "GET /static/assets/qr_preview.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:04:21:07 +0000] "GET /static/assets/images/circlemodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:04:21:07 +0000] "GET /static/assets/images/squaremodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
2021-12-28 16:01:29,004 (INFO) app_qr.py:qr => IP: 127.0.0.1 GET /qr-code page
127.0.0.1 - - [28/Dec/2021:16:01:29 +0000] "GET /qr-code HTTP/1.1" 200 25095 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:29 +0000] "GET /static/js/qr.js HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:29 +0000] "GET /static/assets/images/gappedmodule.png HTTP/1.1" 200 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:29 +0000] "GET /static/assets/images/circlemodule.png HTTP/1.1" 200 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:29 +0000] "GET /static/assets/images/roundedmodule.png HTTP/1.1" 200 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:29 +0000] "GET /static/assets/images/verticalbarmodule.png HTTP/1.1" 200 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:29 +0000] "GET /static/assets/images/horizontalbarmodule.png HTTP/1.1" 200 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:29 +0000] "GET /static/assets/qr_preview.png HTTP/1.1" 200 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:30 +0000] "GET /static/assets/images/squaremodule.png HTTP/1.1" 200 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
2021-12-28 16:01:36,392 (INFO) app_qr.py:qr => IP: 127.0.0.1 POST /qr-code page
127.0.0.1 - - [28/Dec/2021:16:01:36 +0000] "POST /qr-code HTTP/1.1" 200 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
2021-12-28 16:01:49,945 (INFO) app_qr.py:qr => IP: 127.0.0.1 GET /qr-code page
127.0.0.1 - - [28/Dec/2021:16:01:49 +0000] "GET /qr-code HTTP/1.1" 200 25095 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:50 +0000] "GET /static/js/qr.js HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:50 +0000] "GET /static/assets/images/squaremodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:50 +0000] "GET /static/assets/images/gappedmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:50 +0000] "GET /static/assets/images/circlemodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:50 +0000] "GET /static/assets/images/roundedmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:50 +0000] "GET /static/assets/images/verticalbarmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:50 +0000] "GET /static/assets/qr_preview.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:01:50 +0000] "GET /static/assets/images/horizontalbarmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
2021-12-28 16:01:57,785 (INFO) app_qr.py:qr => IP: 127.0.0.1 POST /qr-code page
127.0.0.1 - - [28/Dec/2021:16:01:58 +0000] "POST /qr-code HTTP/1.1" 200 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
2021-12-28 16:02:08,829 (INFO) app_qr.py:qr => IP: 127.0.0.1 GET /qr-code page
127.0.0.1 - - [28/Dec/2021:16:02:08 +0000] "GET /qr-code HTTP/1.1" 200 25095 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:02:08 +0000] "GET /static/assets/images/squaremodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:02:08 +0000] "GET /static/assets/images/gappedmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:02:08 +0000] "GET /static/js/qr.js HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:02:08 +0000] "GET /static/assets/images/roundedmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:02:08 +0000] "GET /static/assets/images/circlemodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:02:08 +0000] "GET /static/assets/images/verticalbarmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:02:09 +0000] "GET /static/assets/images/horizontalbarmodule.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:02:09 +0000] "GET /static/assets/qr_preview.png HTTP/1.1" 304 0 "http://localhost:8080/qr-code" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:10:24 +0000] "GET /qr-code/login HTTP/1.1" 200 2747 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:14:34 +0000] "GET /qr-code/login HTTP/1.1" 200 5860 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:14:34 +0000] "GET /qr-code/signin.css HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:14:34 +0000] "GET /docs/5.0/dist/css/bootstrap.min.css HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:14:34 +0000] "GET /static/js/qr.js HTTP/1.1" 304 0 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:14:34 +0000] "GET /docs/5.0/assets/brand/bootstrap-logo.svg HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:14:34 +0000] "GET /qr-code/signin.css HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:15:35 +0000] "GET /qr-code/login HTTP/1.1" 200 2896 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:15:36 +0000] "GET /docs/5.0/dist/css/bootstrap.min.css HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:15:36 +0000] "GET /docs/5.0/assets/brand/bootstrap-logo.svg HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:15:36 +0000] "GET /qr-code/signin.css HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:15:36 +0000] "GET /docs/5.0/assets/img/favicons/favicon.ico HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:15:36 +0000] "GET /docs/5.0/assets/img/favicons/favicon-32x32.png HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:15:36 +0000] "GET /docs/5.0/assets/img/favicons/favicon-16x16.png HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:15:38 +0000] "GET /docs/5.0/assets/img/favicons/manifest.json HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:18:07 +0000] "GET /qr-code/login HTTP/1.1" 200 3506 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:18:07 +0000] "GET /qr-code/signin.css HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:18:10 +0000] "GET /docs/5.0/assets/img/favicons/manifest.json HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:25:25 +0000] "GET /qr-code/login HTTP/1.1" 200 3462 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:25:28 +0000] "GET /docs/5.0/assets/img/favicons/manifest.json HTTP/1.1" 404 232 "http://localhost:8080/qr-code/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:26:27 +0000] "GET /qr-code/login HTTP/1.1" 200 2813 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:26:30 +0000] "GET /qr-code/login HTTP/1.1" 200 2813 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:32:53 +0000] "GET /qr-code/register HTTP/1.1" 404 232 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:33:13 +0000] "GET /qr-code/register HTTP/1.1" 404 232 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:33:15 +0000] "GET /qr-code/register HTTP/1.1" 404 232 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:33:44 +0000] "GET /qr-code/register HTTP/1.1" 200 3487 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:34:31 +0000] "GET /qr-code/register HTTP/1.1" 200 3425 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:35:18 +0000] "GET /qr-code/register HTTP/1.1" 200 3411 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:35:54 +0000] "GET /qr-code/register HTTP/1.1" 200 3411 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:36:07 +0000] "GET /qr-code/register HTTP/1.1" 200 3426 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:49:42 +0000] "GET /qr-code/register HTTP/1.1" 200 4694 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:53:04 +0000] "GET /qr-code/register HTTP/1.1" 200 4828 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:53:59 +0000] "GET /qr-code/register HTTP/1.1" 200 4838 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:54:14 +0000] "GET /qr-code/register HTTP/1.1" 200 4838 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:54:26 +0000] "GET /qr-code/register? HTTP/1.1" 200 4838 "http://localhost:8080/qr-code/register" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:54:58 +0000] "GET /qr-code/register? HTTP/1.1" 200 4659 "http://localhost:8080/qr-code/register" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:55:48 +0000] "GET /qr-code/register? HTTP/1.1" 200 4630 "http://localhost:8080/qr-code/register" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:56:00 +0000] "GET /qr-code/register HTTP/1.1" 200 4630 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:56:06 +0000] "POST /qr-code/register HTTP/1.1" 405 178 "http://localhost:8080/qr-code/register" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:16:56:44 +0000] "POST /qr-code/register HTTP/1.1" 405 178 "http://localhost:8080/qr-code/register" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:18:03:56 +0000] "GET /qr-code/register HTTP/1.1" 200 4971 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:18:04:08 +0000] "POST /qr-code/register HTTP/1.1" 405 178 "http://localhost:8080/qr-code/register" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:18:04:27 +0000] "GET /qr-code/register HTTP/1.1" 200 4971 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:18:04:42 +0000] "POST /qr-code/register HTTP/1.1" 405 178 "http://localhost:8080/qr-code/register" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:18:05:40 +0000] "GET /qr-code/register HTTP/1.1" 200 4939 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
127.0.0.1 - - [28/Dec/2021:18:05:55 +0000] "POST /qr-code/register HTTP/1.1" 405 178 "http://localhost:8080/qr-code/register" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
2021-12-28 18:06:52,439 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:06:52,445 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:06:52,447 (INFO) _internal.py:_log =>  * Restarting with stat
2021-12-28 18:06:53,880 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:06:53,888 (WARNING) _internal.py:_log =>  * Debugger is active!
2021-12-28 18:06:53,891 (INFO) _internal.py:_log =>  * Debugger PIN: 106-436-927
2021-12-28 18:07:25,470 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:07:25] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:07:25,472 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:07:25] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:07:36,743 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:07:36] "[31m[1mPOST /qr-code/register HTTP/1.1[0m" 405 -
2021-12-28 18:19:03,262 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:19:03,270 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:19:03,273 (INFO) _internal.py:_log =>  * Restarting with stat
2021-12-28 18:19:04,610 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:19:04,615 (WARNING) _internal.py:_log =>  * Debugger is active!
2021-12-28 18:19:04,619 (INFO) _internal.py:_log =>  * Debugger PIN: 106-436-927
2021-12-28 18:19:33,253 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:19:33] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:20:22,415 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:20:22] "[31m[1mPOST /qr-code/register HTTP/1.1[0m" 405 -
2021-12-28 18:20:46,312 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:20:46] "[31m[1mPOST /qr-code/register HTTP/1.1[0m" 405 -
2021-12-28 18:21:02,867 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:21:02,875 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:21:14,281 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:21:14] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:21:28,440 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:21:28] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:21:36,076 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:21:36] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:21:48,887 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:21:48] "[31m[1mPOST /qr-code/register HTTP/1.1[0m" 405 -
2021-12-28 18:22:08,043 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:22:08,052 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:23:10,587 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:23:10] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:26:36,825 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:26:36] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:26:40,801 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:26:40,806 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:26:58,497 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:26:58] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:27:24,182 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:27:24] "[31m[1mPOST /qr-code/register HTTP/1.1[0m" 405 -
2021-12-28 18:28:11,264 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:28:11,270 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:28:30,576 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:28:30,582 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:28:53,445 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:28:53,452 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:29:05,129 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:29:05] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:29:15,307 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:29:15] "[31m[1mPOST /qr-code/register HTTP/1.1[0m" 405 -
2021-12-28 18:29:46,349 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:29:46] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:29:57,221 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:29:57] "[31m[1mPOST /qr-code/register HTTP/1.1[0m" 405 -
2021-12-28 18:30:13,283 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:30:13,289 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:30:21,552 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:30:21] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:30:21,953 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:30:21] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2021-12-28 18:30:27,544 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:30:27] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:30:49,555 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:30:49] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:30:52,240 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:30:52] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2021-12-28 18:30:54,328 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:30:54] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:31:06,849 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:31:06] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:31:29,369 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:31:29] "[31m[1mPOST /qr-code/register HTTP/1.1[0m" 405 -
2021-12-28 18:33:01,126 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:33:01,132 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:33:15,696 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:33:15] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:33:22,649 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1701, in make_response
    raise TypeError(
TypeError: The view function for 'app_login.register' did not return a valid response. The function either returned None or ended without a return statement.
2021-12-28 18:33:22,654 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:33:22] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 18:34:01,435 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:34:01,441 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:34:06,307 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:34:06] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:34:08,807 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:34:08] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:34:23,621 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1701, in make_response
    raise TypeError(
TypeError: The view function for 'app_login.register' did not return a valid response. The function either returned None or ended without a return statement.
2021-12-28 18:34:23,625 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:34:23] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 18:34:52,495 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:34:52,500 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:34:58,561 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:34:58] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:45:15,327 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:45:15,332 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:45:53,775 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:45:53] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2021-12-28 18:45:56,805 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:45:56] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:48:46,044 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:48:46,052 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:48:54,862 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:48:54] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 18:49:31,932 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 17, in register
    obj = json.loads(req)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not dict
2021-12-28 18:49:31,949 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:49:31] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 18:50:14,666 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 17, in register
    obj = json.loads(req)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not dict
2021-12-28 18:50:14,676 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:50:14] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 18:52:06,093 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 18:52:06,101 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 18:52:34,205 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 18:52:34] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 19:19:03,427 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 19:19:03,434 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 19:19:18,139 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:19:18] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 19:19:51,685 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in register
    exec_query(req)
  File "/app/app_login.py", line 32, in exec_query
    con.execute('INSERT INTO user (  company_name, first_name, last_name, email, password) VALUES ( values["companyName"], values["firstName"], values["lastName"], values["email"], values["password"])')
sqlite3.OperationalError: near "values": syntax error
2021-12-28 19:19:51,704 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:19:51] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 19:28:13,398 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 19:28:13,403 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 19:31:57,109 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 19:31:57,114 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 19:32:11,759 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:32:11] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 19:33:05,565 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in register
    exec_query(req)
  File "/app/app_login.py", line 39, in exec_query
    return render_template("success.html",msg = msg)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 148, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 1068, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/usr/local/lib/python3.8/dist-packages/jinja2/loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 59, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 95, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: success.html
2021-12-28 19:33:05,586 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:33:05] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 19:33:30,607 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 19:33:30,616 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 19:33:51,517 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:33:51] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 19:33:55,447 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:33:55] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 19:34:34,433 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in register
    exec_query(req)
  File "/app/app_login.py", line 31, in exec_query
    cur.execute("INSERT INTO user (company_name, first_name, last_name, email, password) values (?,?,?,?,?)",(values["companyName"], values["firstName"], values["lastName"], values["email"], values["password"]))
sqlite3.OperationalError: no such table: user
2021-12-28 19:34:34,453 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:34:34] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 19:35:42,229 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 19:35:42,235 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 19:35:51,847 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:35:51] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 19:36:16,974 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in register
    exec_query(req)
  File "/app/app_login.py", line 45, in exec_query
    return render_template("success.html",msg = msg)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 148, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 1068, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/usr/local/lib/python3.8/dist-packages/jinja2/loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 59, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 95, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: success.html
2021-12-28 19:36:16,997 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:36:16] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 19:37:10,384 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 19:37:10,392 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 19:37:17,246 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:37:17] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 19:38:16,975 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in register
    exec_query(req)
  File "/app/app_login.py", line 45, in exec_query
    return render_template("success.html",msg = msg)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 148, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 1068, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/usr/local/lib/python3.8/dist-packages/jinja2/loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 59, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 95, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: success.html
2021-12-28 19:38:16,991 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:38:16] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 19:41:17,851 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 19:41:17,861 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 19:41:24,056 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:41:24] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 19:41:24,114 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 19:41:24] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 20:37:25,862 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 20:37:25,870 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 20:37:31,254 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 20:37:31] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 20:37:37,178 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 20:37:37] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 20:38:17,199 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 22, in register
    exec_query(req)
  File "/app/app_login.py", line 52, in exec_query
    return render_template("success.html",msg = msg)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 148, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 1068, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "/usr/local/lib/python3.8/dist-packages/jinja2/environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/usr/local/lib/python3.8/dist-packages/jinja2/loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 59, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.8/dist-packages/flask/templating.py", line 95, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: success.html
2021-12-28 20:38:17,212 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 20:38:17] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 21:19:37,408 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 21:19:37,414 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 21:19:53,378 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 21:19:53] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 21:34:09,825 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 21:34:09,836 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 21:34:15,936 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 21:34:15] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 21:36:33,014 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 21:36:33] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 21:47:03,522 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 21:47:03] "POST /qr-code/register HTTP/1.1" 200 -
2021-12-28 21:50:21,730 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 21:50:21] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:01:29,510 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:01:29,515 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:01:40,171 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:01:40] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:01:40,229 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:01:40] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:02:15,557 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 28, in register
    return jsonify(result)
  File "/usr/local/lib/python3.8/dist-packages/flask/json/__init__.py", line 355, in jsonify
    f"{dumps(data, indent=indent, separators=separators)}\n",
  File "/usr/local/lib/python3.8/dist-packages/flask/json/__init__.py", line 133, in dumps
    rv = _json.dumps(obj, **kwargs)
  File "/usr/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.8/dist-packages/flask/json/__init__.py", line 57, in default
    return super().default(o)
  File "/usr/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type type is not JSON serializable
2021-12-28 22:02:15,573 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:02:15] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 22:04:48,770 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 28, in register
    return jsonify(result)
  File "/usr/local/lib/python3.8/dist-packages/flask/json/__init__.py", line 355, in jsonify
    f"{dumps(data, indent=indent, separators=separators)}\n",
  File "/usr/local/lib/python3.8/dist-packages/flask/json/__init__.py", line 133, in dumps
    rv = _json.dumps(obj, **kwargs)
  File "/usr/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.8/dist-packages/flask/json/__init__.py", line 57, in default
    return super().default(o)
  File "/usr/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type type is not JSON serializable
2021-12-28 22:04:48,781 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:04:48] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-28 22:07:40,210 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:07:40,215 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:07:49,517 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:07:49] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:08:07,406 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:08:07] "POST /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:18:26,338 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:18:26,346 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:18:38,686 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:18:38] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:19:44,778 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:19:44,784 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:19:54,247 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:19:54] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:21:19,608 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:21:19,617 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:21:24,544 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:21:24] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:22:00,579 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:22:00,585 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:22:06,996 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:22:06] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:22:28,062 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in login
    values = (req["email"], sha256_crypt.encrypt(req["password"]))
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/decor.py", line 191, in wrapper
    return func(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/ifc.py", line 154, in encrypt
    return cls.hash(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 777, in hash
    validate_secret(secret)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 122, in validate_secret
    raise exc.ExpectedStringError(secret, "secret")
TypeError: secret must be unicode or bytes, not dict
2021-12-28 22:22:28,073 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:22:28] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:23:00,663 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:23:00,670 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:23:11,205 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:23:11] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:23:25,572 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in login
    values = (req["email"], sha256_crypt.encrypt(req["password"]))
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/decor.py", line 191, in wrapper
    return func(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/ifc.py", line 154, in encrypt
    return cls.hash(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 777, in hash
    validate_secret(secret)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 122, in validate_secret
    raise exc.ExpectedStringError(secret, "secret")
TypeError: secret must be unicode or bytes, not dict
2021-12-28 22:23:25,591 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:23:25] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:23:48,980 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:23:48] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:24:08,129 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:24:08] "POST /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:24:46,403 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:24:46] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:24:59,400 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in login
    values = (req["email"], sha256_crypt.encrypt(req["password"]))
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/decor.py", line 191, in wrapper
    return func(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/ifc.py", line 154, in encrypt
    return cls.hash(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 777, in hash
    validate_secret(secret)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 122, in validate_secret
    raise exc.ExpectedStringError(secret, "secret")
TypeError: secret must be unicode or bytes, not dict
2021-12-28 22:24:59,410 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:24:59] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:25:15,579 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in login
    values = (req["email"], sha256_crypt.encrypt(req["password"]))
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/decor.py", line 191, in wrapper
    return func(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/ifc.py", line 154, in encrypt
    return cls.hash(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 777, in hash
    validate_secret(secret)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 122, in validate_secret
    raise exc.ExpectedStringError(secret, "secret")
TypeError: secret must be unicode or bytes, not dict
2021-12-28 22:25:15,590 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:25:15] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:25:38,861 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 21, in login
    values = (req["email"], sha256_crypt.encrypt(req["password"]))
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/decor.py", line 191, in wrapper
    return func(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/ifc.py", line 154, in encrypt
    return cls.hash(*args, **kwds)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 777, in hash
    validate_secret(secret)
  File "/usr/local/lib/python3.8/dist-packages/passlib/utils/handlers.py", line 122, in validate_secret
    raise exc.ExpectedStringError(secret, "secret")
TypeError: secret must be unicode or bytes, not dict
2021-12-28 22:25:38,866 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:25:38] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:26:15,786 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:26:15,791 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:27:03,988 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:27:03] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:27:28,409 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:27:28] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:28:51,311 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:28:51] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:28:56,370 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:28:56,376 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:29:03,667 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:29:03] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:29:24,628 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:29:24] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:32:11,152 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:32:11] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:34:42,772 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:34:42,780 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:34:58,275 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:34:58] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:35:45,291 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:35:45,300 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:35:56,940 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:35:56] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:36:12,514 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 22, in login
    result = exec_query(query, values)
  File "/app/app_login.py", line 51, in exec_query
    cur.execute(query, values)
sqlite3.ProgrammingError: Incorrect number of bindings supplied. The current statement uses 1, and there are 22 supplied.
2021-12-28 22:36:12,530 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:36:12] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:36:54,749 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 22, in login
    result = exec_query(query, values)
  File "/app/app_login.py", line 51, in exec_query
    cur.execute(query, values)
sqlite3.ProgrammingError: Incorrect number of bindings supplied. The current statement uses 1, and there are 22 supplied.
2021-12-28 22:36:54,758 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:36:54] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:38:03,114 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:38:03,120 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:38:19,564 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:38:19] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:39:05,604 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:39:05] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:39:28,321 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:39:28,326 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:39:40,024 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:39:40] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:46:48,992 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:46:48] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:46:55,899 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:46:55,903 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:47:06,240 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:47:06] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:47:30,699 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:47:30] "POST /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:48:01,945 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:48:01] "POST /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:48:29,553 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:48:29] "POST /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:49:16,978 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:49:16] "POST /qr-code/register HTTP/1.1" 200 -
2021-12-28 22:49:25,807 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:49:25] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:49:57,253 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 23, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-28 22:49:57,271 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:49:57] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:50:54,353 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-28 22:50:54,360 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-28 22:51:20,094 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:51:20] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-28 22:51:40,459 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 23, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-28 22:51:40,477 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:51:40] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:52:21,293 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 23, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-28 22:52:21,301 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:52:21] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-28 22:52:40,963 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 23, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-28 22:52:40,973 (INFO) _internal.py:_log => 127.0.0.1 - - [28/Dec/2021 22:52:40] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-29 01:03:14,482 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:03:14,489 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:03:25,788 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:03:25] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:03:55,961 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 22, in login
    result = exec_query(query, values)
  File "/app/app_login.py", line 62, in exec_query
    return msg
UnboundLocalError: local variable 'msg' referenced before assignment
2021-12-29 01:03:55,976 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:03:55] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-29 01:04:47,948 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:04:47,953 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:04:59,373 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:04:59] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:05:16,761 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 23, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-29 01:05:16,779 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:05:16] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-29 01:06:16,036 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:06:16,045 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:06:44,805 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:06:44] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:07:09,626 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 23, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-29 01:07:09,640 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:07:09] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-29 01:09:32,172 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:09:32,180 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:11:25,133 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:11:25,144 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:13:01,816 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 32, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-29 01:13:01,827 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:13:01] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-29 01:16:14,752 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:16:14,761 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:16:58,690 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 23, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-29 01:16:58,712 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:16:58] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-29 01:18:01,668 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:18:01,678 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:18:10,434 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:18:10] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:19:48,985 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:19:48,994 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:19:57,690 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:19:57] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:20:35,717 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 23, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-29 01:20:35,736 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:20:35] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-29 01:20:57,243 (ERROR) app.py:log_exception => Exception on /qr-code/login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 23, in login
    if result > 0:
TypeError: '>' not supported between instances of 'list' and 'int'
2021-12-29 01:20:57,250 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:20:57] "[35m[1mPOST /qr-code/login HTTP/1.1[0m" 500 -
2021-12-29 01:21:13,079 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:21:13,087 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:23:05,126 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:23:05,132 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:23:21,685 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:23:21] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:25:42,659 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:25:42,666 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:25:55,628 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:25:55] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:26:10,038 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:26:10] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-29 01:29:28,794 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:29:28,800 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:29:39,550 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:29:39] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-29 01:30:39,458 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:30:39] "POST /qr-code/register HTTP/1.1" 200 -
2021-12-29 01:30:44,895 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 37, in register
    result = query_insert(query, values)
  File "/app/app_login.py", line 56, in query_insert
    cur.execute(query, values)
sqlite3.IntegrityError: UNIQUE constraint failed: user.email
2021-12-29 01:30:44,913 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:30:44] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-29 01:34:06,145 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:34:06] "GET /qr-code/register HTTP/1.1" 200 -
2021-12-29 01:34:49,793 (ERROR) app.py:log_exception => Exception on /qr-code/register [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/app/app_login.py", line 37, in register
    result = query_insert(query, values)
  File "/app/app_login.py", line 56, in query_insert
sqlite3.OperationalError: database is locked
2021-12-29 01:34:49,803 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:34:49] "[35m[1mPOST /qr-code/register HTTP/1.1[0m" 500 -
2021-12-29 01:35:07,202 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:35:07,209 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:35:24,980 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:35:24] "POST /qr-code/register HTTP/1.1" 200 -
2021-12-29 01:43:03,794 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:43:03,802 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:43:13,805 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:43:13] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:45:52,742 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:45:52,749 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:46:02,940 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:46:02] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:47:50,238 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:47:50] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:48:04,345 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 01:48:04,350 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 01:48:19,994 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:48:19] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:48:52,195 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:48:52] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:49:17,193 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:49:17] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-29 01:49:29,097 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 01:49:29] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-29 02:13:14,408 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 02:13:14,414 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 02:13:28,797 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:13:28] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 02:13:29,945 (ERROR) app.py:log_exception => Exception on /qr-code/admin/room/main [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
TypeError: index() got an unexpected keyword argument 'room'
2021-12-29 02:13:29,952 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:13:29] "[35m[1mGET /qr-code/admin/room/main HTTP/1.1[0m" 500 -
2021-12-29 02:13:32,818 (ERROR) app.py:log_exception => Exception on /qr-code/admin/room/main [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
TypeError: index() got an unexpected keyword argument 'room'
2021-12-29 02:13:32,821 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:13:32] "[35m[1mGET /qr-code/admin/room/main HTTP/1.1[0m" 500 -
2021-12-29 02:14:06,803 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 02:14:06,808 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 02:14:14,946 (ERROR) app.py:log_exception => Exception on /qr-code/admin/room/main [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
TypeError: index() got an unexpected keyword argument 'room'
2021-12-29 02:14:14,956 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:14:14] "[35m[1mGET /qr-code/admin/room/main HTTP/1.1[0m" 500 -
2021-12-29 02:15:13,188 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 02:15:13,195 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 02:15:21,051 (ERROR) app.py:log_exception => Exception on /qr-code/admin/room/main [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
TypeError: index() got an unexpected keyword argument 'room'
2021-12-29 02:15:21,061 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:15:21] "[35m[1mGET /qr-code/admin/room/main HTTP/1.1[0m" 500 -
2021-12-29 02:16:09,770 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 02:16:09,779 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 02:16:13,008 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:16:13] "GET /qr-code/admin/room/main HTTP/1.1" 200 -
2021-12-29 02:16:13,574 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:16:13] "GET /qr-code/admin/room/main HTTP/1.1" 200 -
2021-12-29 02:16:13,816 (ERROR) server.py:_log_error_once => The WebSocket transport is not available, you must install a WebSocket server that is compatible with your async mode to enable it. See the documentation for details. (further occurrences of this error will be logged with level INFO)
2021-12-29 02:16:13,822 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:16:13] "GET /socket.io/?EIO=4&transport=polling&t=Nu3-K3R HTTP/1.1" 200 -
2021-12-29 02:16:17,008 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:16:17] "POST /socket.io/?EIO=4&transport=polling&t=Nu3-K5d&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:16:17,369 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:16:17] "GET /socket.io/?EIO=4&transport=polling&t=Nu3-K5e&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:16:38,849 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:16:38] "GET /socket.io/?EIO=4&transport=polling&t=Nu3-Kyh&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:16:42,153 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:16:42] "POST /socket.io/?EIO=4&transport=polling&t=Nu3-QCK&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:17:07,180 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:17:07] "GET /socket.io/?EIO=4&transport=polling&t=Nu3-QCL&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:17:07,318 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:17:07] "POST /socket.io/?EIO=4&transport=polling&t=Nu3-X7N&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:17:32,343 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:17:32] "GET /socket.io/?EIO=4&transport=polling&t=Nu3-X7O&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:17:35,402 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:17:35] "POST /socket.io/?EIO=4&transport=polling&t=Nu3-dG5&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:18:00,409 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:18:00] "GET /socket.io/?EIO=4&transport=polling&t=Nu3-dG6&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:18:00,511 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:18:00] "POST /socket.io/?EIO=4&transport=polling&t=Nu3-k6a&sid=MXbzqJ5443SlhO5rAAAA HTTP/1.1" 200 -
2021-12-29 02:18:24,392 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 02:18:24,401 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 02:18:30,411 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:18:30] "GET /qr-code/admin/room/main HTTP/1.1" 200 -
2021-12-29 02:18:51,214 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:18:51] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-29 02:18:55,900 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:18:55] "GET /qr-code/admin/room/main HTTP/1.1" 200 -
2021-12-29 02:18:58,443 (ERROR) server.py:_log_error_once => The WebSocket transport is not available, you must install a WebSocket server that is compatible with your async mode to enable it. See the documentation for details. (further occurrences of this error will be logged with level INFO)
2021-12-29 02:18:58,449 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:18:58] "GET /socket.io/?EIO=4&transport=polling&t=Nu3-xgf HTTP/1.1" 200 -
2021-12-29 02:19:05,066 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:19:05] "POST /socket.io/?EIO=4&transport=polling&t=Nu3-yHW&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:19:05,169 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:19:05] "GET /socket.io/?EIO=4&transport=polling&t=Nu3-yHX&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:19:23,475 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:19:23] "GET /socket.io/?EIO=4&transport=polling&t=Nu3-zwR&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:19:26,997 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:19:26] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_2On&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:19:52,024 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:19:52] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_2On.0&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:19:52,126 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:19:52] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_9MZ&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:20:17,153 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:20:17] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_9MZ.0&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:20:20,334 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:20:20] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_FVR&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:20:45,348 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:20:45] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_FVS&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:20:45,464 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:20:45] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_MNu&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:21:10,491 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:21:10] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_MNv&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:21:13,906 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:21:13] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_SWX&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:21:38,933 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:21:38] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_SWY&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:21:39,049 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:21:39] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_ZT9&sid=KUQoOv0Yv_eu94fEAAAA HTTP/1.1" 200 -
2021-12-29 02:22:08,988 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 02:22:08,994 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 02:22:12,788 (ERROR) server.py:_log_error_once => The WebSocket transport is not available, you must install a WebSocket server that is compatible with your async mode to enable it. See the documentation for details. (further occurrences of this error will be logged with level INFO)
2021-12-29 02:22:12,797 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:12] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_gXd HTTP/1.1" 200 -
2021-12-29 02:22:13,200 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:13] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_hkJ&sid=PqTqAScIiDEwN4a3AAAA HTTP/1.1" 200 -
2021-12-29 02:22:15,952 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:15] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_hkM&sid=PqTqAScIiDEwN4a3AAAA HTTP/1.1" 200 -
2021-12-29 02:22:16,979 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:16] "GET /qr-code/admin/room/main HTTP/1.1" 200 -
2021-12-29 02:22:17,285 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:17] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_hyq&sid=PqTqAScIiDEwN4a3AAAA HTTP/1.1" 200 -
2021-12-29 02:22:20,135 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:20] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_imA HTTP/1.1" 200 -
2021-12-29 02:22:20,923 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:20] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_jWo&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:22:23,119 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:23] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_jWp&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:22:45,162 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:45] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_kFr&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:22:45,267 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:22:45] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_pdq&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:23:10,294 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:23:10] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_pdr&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:23:13,518 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:23:13] "POST /socket.io/?EIO=4&transport=polling&t=Nu3_voP&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:23:38,543 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:23:38] "GET /socket.io/?EIO=4&transport=polling&t=Nu3_voQ&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:23:38,647 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:23:38] "POST /socket.io/?EIO=4&transport=polling&t=Nu400fv&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:24:03,674 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:24:03] "GET /socket.io/?EIO=4&transport=polling&t=Nu400fw&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:24:06,748 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:24:06] "POST /socket.io/?EIO=4&transport=polling&t=Nu406oh&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:24:31,772 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:24:31] "GET /socket.io/?EIO=4&transport=polling&t=Nu406oi&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:24:31,885 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:24:31] "POST /socket.io/?EIO=4&transport=polling&t=Nu40Dfh&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:24:56,910 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:24:56] "GET /socket.io/?EIO=4&transport=polling&t=Nu40Dfi&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:25:00,119 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:25:00] "POST /socket.io/?EIO=4&transport=polling&t=Nu40JoT&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:25:25,143 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:25:25] "GET /socket.io/?EIO=4&transport=polling&t=Nu40JoU&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:25:25,306 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:25:25] "POST /socket.io/?EIO=4&transport=polling&t=Nu40QiO&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:25:50,332 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:25:50] "GET /socket.io/?EIO=4&transport=polling&t=Nu40QiP&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:25:53,560 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:25:53] "POST /socket.io/?EIO=4&transport=polling&t=Nu40WrE&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:26:18,582 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:26:18] "GET /socket.io/?EIO=4&transport=polling&t=Nu40WrF&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:26:18,697 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:26:18] "POST /socket.io/?EIO=4&transport=polling&t=Nu40dki&sid=JjLj6KeFiniWG2URAAAC HTTP/1.1" 200 -
2021-12-29 02:26:46,377 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 02:26:46,384 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 02:27:02,695 (ERROR) server.py:_log_error_once => The WebSocket transport is not available, you must install a WebSocket server that is compatible with your async mode to enable it. See the documentation for details. (further occurrences of this error will be logged with level INFO)
2021-12-29 02:27:02,705 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:02] "GET /socket.io/?EIO=4&transport=polling&t=Nu40nqJ HTTP/1.1" 200 -
2021-12-29 02:27:06,411 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:06] "POST /socket.io/?EIO=4&transport=polling&t=Nu40oa4&sid=S-1y0XSsLNpzUBoaAAAA HTTP/1.1" 200 -
2021-12-29 02:27:06,412 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:06] "GET /socket.io/?EIO=4&transport=polling&t=Nu40oa5&sid=S-1y0XSsLNpzUBoaAAAA HTTP/1.1" 200 -
2021-12-29 02:27:13,179 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:13] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 02:27:20,462 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:20] "GET /qr-code/logout HTTP/1.1" 200 -
2021-12-29 02:27:27,087 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:27] "[32mGET /qr-code/admin/room/main HTTP/1.1[0m" 302 -
2021-12-29 02:27:27,446 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:27] "POST /socket.io/?EIO=4&transport=polling&t=Nu40tjC&sid=S-1y0XSsLNpzUBoaAAAA HTTP/1.1" 200 -
2021-12-29 02:27:27,447 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:27] "GET /socket.io/?EIO=4&transport=polling&t=Nu40pV8&sid=S-1y0XSsLNpzUBoaAAAA HTTP/1.1" 200 -
2021-12-29 02:27:30,041 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:30] "GET /qr-code/login HTTP/1.1" 200 -
2021-12-29 02:27:59,313 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:27:59] "POST /qr-code/login HTTP/1.1" 200 -
2021-12-29 02:28:06,461 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:28:06] "GET /qr-code/admin/room/main HTTP/1.1" 200 -
2021-12-29 02:28:09,052 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:28:09] "GET /socket.io/?EIO=4&transport=polling&t=Nu4126G HTTP/1.1" 200 -
2021-12-29 02:28:12,469 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:28:12] "POST /socket.io/?EIO=4&transport=polling&t=Nu412iY&sid=ZJw5HfW6dBWwsA2JAAAC HTTP/1.1" 200 -
2021-12-29 02:28:12,958 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:28:12] "GET /socket.io/?EIO=4&transport=polling&t=Nu412iZ&sid=ZJw5HfW6dBWwsA2JAAAC HTTP/1.1" 200 -
2021-12-29 02:28:34,062 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:28:34] "GET /socket.io/?EIO=4&transport=polling&t=Nu413fr&sid=ZJw5HfW6dBWwsA2JAAAC HTTP/1.1" 200 -
2021-12-29 02:28:37,276 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:28:37] "POST /socket.io/?EIO=4&transport=polling&t=Nu418pS&sid=ZJw5HfW6dBWwsA2JAAAC HTTP/1.1" 200 -
2021-12-29 02:29:02,300 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:29:02] "GET /socket.io/?EIO=4&transport=polling&t=Nu418pT&sid=ZJw5HfW6dBWwsA2JAAAC HTTP/1.1" 200 -
2021-12-29 02:29:02,407 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:29:02] "POST /socket.io/?EIO=4&transport=polling&t=Nu41Fif&sid=ZJw5HfW6dBWwsA2JAAAC HTTP/1.1" 200 -
2021-12-29 02:29:15,295 (WARNING) _internal.py:_log => WebSocket transport not available. Install simple-websocket for improved performance.
2021-12-29 02:29:15,303 (INFO) _internal.py:_log =>  * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
2021-12-29 02:29:22,725 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:29:22] "GET /qr-code/admin/room/main HTTP/1.1" 200 -
2021-12-29 02:29:25,830 (ERROR) server.py:_log_error_once => The WebSocket transport is not available, you must install a WebSocket server that is compatible with your async mode to enable it. See the documentation for details. (further occurrences of this error will be logged with level INFO)
2021-12-29 02:29:25,837 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:29:25] "GET /socket.io/?EIO=4&transport=polling&t=Nu41Kjl HTTP/1.1" 200 -
2021-12-29 02:29:26,287 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:29:26] "GET /qr-code/logout HTTP/1.1" 200 -
2021-12-29 02:29:36,398 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:29:36] "[32mGET /qr-code/admin/room/main HTTP/1.1[0m" 302 -
2021-12-29 02:29:36,875 (INFO) _internal.py:_log => 127.0.0.1 - - [29/Dec/2021 02:29:36] "GET /qr-code/login HTTP/1.1" 200 -
127.0.0.1 - - [29/Dec/2021:02:33:58 +0000] "GET /qr-code/logout HTTP/1.1" 200 4080 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36"
